examples/retinanet/util.py

```diff
@@ -0,0 +1,506 @@
+from jax import numpy as jnp
+
+import jax
+import numpy as np
+import tensorflow as tf
+
+CATEGORY_MAP = {
```

Comment:
isn't this list already contained in `coco_eval.py` ?

Reply:
Good point. It's actually not even used, so I'm going to remove it.